src/scss/skin-modern/_mixins.scss

```diff
@@ -28,7 +28,7 @@
 
   &.#{$prefix}-hidden {
     opacity: 0;
-    transition: opacity $duration, visibility $duration;
+    transition: opacity $duration $duration;
```

Comment:
Is this valid CSS? I don't think this are the correct properties for the `transition`